yt/docker/local/run_local_cluster.sh

```diff
@@ -13,7 +13,7 @@ script_name=$0
 proxy_port=8000
 docker_hostname=localhost
 interface_port=8001
-yt_version=stable
+yt_version=dev
```

Comment:
not sure about this change, but `stable` seems pretty outdated, it doesn't even support the native client setting

Reply:
@savnadya – what do you think about this change? (stable -> dev)

Reply:
I don't think it's a good idea. `dev` image is updated after each commit, and it can break easily. You can just push a new stable version of the image.

Reply:
I doubt I have access to push a new image - who can do this?